web/client/index.html

```diff
@@ -74,8 +77,10 @@
         </style>
     </head>
     <body>
+        <a href="https://github.com/geosolutions-it/MapStore2"><img style="position: absolute; top: 0; right: 0; border: 0; z-index:100; " src="https://camo.githubusercontent.com/e7bbb0521b397edbd5fe43e7f760759336b5e05f/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677265656e5f3030373230302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png"></a>
```

Comment:
It is possible, but it require a little more work with no gain (only a usage sample IMHO). I suggest to use this now and open a minor priority issue to fix. Let me know if it is ok for you.